README.md

```diff
@@ -96,6 +96,37 @@ const router = require('find-my-way')({
 })
 ```
 
+You can assign a `buildPrettyPrint` function to sanitize a route's `store` object to use with the `prettyPrint` functions. This function should accept a single object and return an object.
```

Comment:
`buildPrettyPrint` or `buildPrettyMeta`?

Reply:
Whoops, good catch! Should be `buildPrettyMeta` (fixed now)